generate/src/contract.rs

```diff
@@ -61,11 +63,24 @@ impl Context {
             artifact,
             runtime_crate,
             contract_name,
```

Comment:
My motivation here was that as `Args` grows, we don't have to keep adding new fields to `Context` - but I agree, it will probably be cleaner than just keeping the whole `Args` around.

The `runtime_crate` and `contract_name` come from the `Args` but have been transformed to `Ident`s so they can be `quote!`-ed.

Also, in the works is the linking PR which will add more arguments (linking options).